src/msw/webrequest_winhttp.cpp

```diff
@@ -214,6 +214,38 @@ static wxString wxWinHTTPQueryHeaderString(HINTERNET hRequest, DWORD dwInfoLevel
     return result;
 }
 
+static std::vector<wxString> wxWinHTTPQueryAllHeaderStrings(HINTERNET hRequest, DWORD dwInfoLevel,
```

Comment:
Could this be refactored to reuse the existing `wxWinHTTPQueryHeaderString()` by adding an optional `DWORD* nextIndex = WINHTTP_NO_HEADER_INDEX` parameter to it and passing it `&nextIndex` when calling it from here?

The code dealing with buffers is sufficiently non trivial for me not to want to duplicate it...